### DISCUSSION TITLE
add a new property to NormalEntry.j2.tex

### DISCUSSION THREAD

**Discussion by majaneubauer** [rendercv/rendercv#142] [Q&A]
hi everyone,

I was wondering whether it would be possible to add a new property to the NormalEntry.j2.tex file. Specifically I am trying to have my name property, then a text property (this would be the new one) and then my highlights property. In the .yaml file it would look something ike this:

```
relevant coursework:
      - name: Course Name
        start_date: 2023-06-12
        end_date: 2023-07-07
        text: some text 
        highlights:
          - first bullet point
 ```

I have tried to modify my NormalEntry.j2.tex file in this way, but with no luck

```
((* if entry.date_string or entry.location *))
\begin{twocolentry}{
    <<date_and_location_strings|join("\n\n")>>
}
    \textbf{<<entry.name>>}
    ((* if entry.text *))
    \par <<entry.text>>
    ((* endif *))
    ((* for item in entry.highlights *))
        ((* if loop.first *))
    \begin{highlights}
        ((* endif *))
        \item <<item>>
        ((* if loop.last *))
    \end{highlights}
        ((* endif *))
    ((* endfor *))
\end{twocolentry}
```

I would be grateful for any suggestions. 

Thank you for your help!

**Comment by sinaatalay** [comment]
Hi,

This feature is already supported (see the [docs](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#using-arbitrary-keys)), the example you gave should have worked.

Are you sure you are using the latest RenderCV version? Could you please check with `rendercv --version`?

**Reply by majaneubauer** [reply]
Hi,

I wasn't aware I had to create a custom theme. Everything is working as it should now, thank you very much!

**Reply by sinaatalay** [reply]
You don’t need to create a custom theme; you can just override the built-in themes, as explained [here](https://docs.rendercv.com/user_guide/#overriding-built-in-themes).
